### DISCUSSION TITLE
Pumpkin head

### DISCUSSION THREAD

**Discussion by Gloktober** [winstxnhdw/lc-hax#91] [Ideas]
Hey again ! do you think it would be possible to attach objects to players head ?

**Comment by winstxnhdw** [comment]
Easily, but they won't be able to see and it'd be terribly noisy for them.

**Reply by Gloktober** [reply]
that's the whole point ;) we need a lot of troll options ahah

**Reply by winstxnhdw** [reply]
I'll get it done at a later date. I have to find a way to get the head of the player.

**Reply by Gloktober** [reply]
awesome !

**Reply by winstxnhdw** [reply]
Hey, I've added a really preliminary version of `/pumpkin`. I haven't tested it or anything so let me know how it goes. I reckon the rotations might be off.

**Reply by Gloktober** [reply]
will try right now lol !

**Reply by Gloktober** [reply]
doesn't work , the command seems to execute but nothing happens

**Reply by winstxnhdw** [reply]
AH SHUCKS, I forgot to unlock it. Will push a fix in a minute.

**Reply by Gloktober** [reply]
nevermind ! i just need to build first than i can attach it to someone head

**Reply by winstxnhdw** [reply]
Done. Fixed in latest. You shouldn't have to build it first anymore. Send a screenshot if you can.

**Reply by Gloktober** [reply]
awesome ! as you said the only thing off is the rotation but it's still really funny lol
